doc/sphinx/requirements.txt

```diff
@@ -0,0 +1,13 @@
+altair>=5.0
+breathe>=4.30
+myst-parser[linkify]>=0.14.0
+sphinx-hoverxref>=0.3b1
+sphinx-design
+sphinx>=5.3,<6
+sphinx_rtd_theme
+sphinxcontrib-bibtex==2.5
+sphinxcontrib-katex
+sphinxcontrib-mermaid
+sphinxcontrib-svg2pdfconverter
+sphinxext-altair
+docutils
```

Comment:
Thanks for the additions, @pritkc ! 
Please also add a README file to the doc (or doc/sphinx) subdirectory to explain how to use the requirements file. See [this](https://github.com/CEED/libCEED/tree/main/doc) for reference. Thank you!